src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -562,17 +667,32 @@ public void enableRandomHead() {
     }
   }
 
+  public void enable() {
```

Comment:
could this list be more dynamic, perhaps from the default inmoov plan instead of this enumerated list here?